pages.id/common/nvim.md

```diff
@@ -30,7 +30,7 @@
 
 - Menyimpan (write) berkas, dan keluar:
 
-`<Esc>:wq<Enter>`
+`{{<Esc>ZZ|<Esc>:wq<Enter>}}`
```

Comment:
Inconsistency: On some translations you put `{{<Esc>ZZ|<Esc>:wq<Enter>}}`, while on other you miss the escape sequence `{{ZZ|:wq<Entrée>}}`